controllers/postControllers.js

```diff
@@ -54,7 +65,13 @@ const createPost = async (post, userId) => {
 
 const updatePost = async (postId, data) => {
   try {
-    return await Post.query().patchAndFetchById(postId, data);
+    const post = await Post.query().patchAndFetchById(postId, data);
+    delete post.created_at;
+    delete post.updated_at;
+    delete post.view_count;
```

Comment:
why are you deleting view_count?

Reply:
Because I thought if we want to get comments' information we don't need to get view_count of post. If we want to get view_count of post we can get the post.

Reply:
I thought about it and realized that it's not bad to delete this data to make sure it's not part of the updated data, but I didn't understand what comments you were talking about, it's just updatePost.

Reply:
Oh sorry. It's my mistake... I have copied the code from commentControllers and forgot to delete that row and i didn't see about which part of code you are asking. That's why I thought you are asking about comments. I also think that it's ok but if we show view_count it's also true. I can change my code if you want. And thanks for comment❤️❤️❤️ I'am glad to know your opinion.

Reply:
I think both options are right